README.md

```diff
@@ -62,7 +62,7 @@ $ xinference -H,--host "localhost" \
              --log-level INFO
 ```
 
-To deploy Xinference in a cluster, you need to start an Xinference supervisor on one server and 
+To deploy Xinference in a cluster, you need to start a Xinference supervisor on one server and 
```

Comment:
Since the pronunciation of the individual letter "X" begins with a vowel sound, we use the indefinite article "an".

Reply:
that makes sense